eduroam.md

```diff
@@ -40,98 +53,190 @@ The process of connecting to the network may be different depending on the Linux
 
 - *Identity:* Your RIT email address. i.e. [username]@rit.edu
 - *Authentication:* TLS
-- *Wifi Security:* WPA and WPA2 Enterprise
-- *CA certificate:* The RIT .cer certificate you downloaded earlier.
+- *Wifi Security:* WPA2 Enterprise
+- *CA certificate:* The RIT `.cer` certificate you downloaded earlier.
 - *User Certificate/private key:* The other file you downloaded (**not the .cer file**). Note that, depending on the file you download, the certificate and private key may or may not be separate.
 - *User Key Password:* The password you set for the above file earlier.
 
+## GUI Setup
+
+This section will work with both `.pem` and `.p12` certificates and keys.
 
-### NetworkManager (GUI) Setup
+### NetworkManager Configuration
 
-1. To connect click the wifi logo in the status bar, and then select 'eduroam' ![](/assets/img/eduroam/open-networkmanager.png)
+1. To connect click the wifi logo in the status bar, and then select `eduroam`. ![](/assets/img/eduroam/open-networkmanager.png)
 
 2. Enter the configuration details in the window that appears. If the password field turns red it means the password is incorrect. ![](/assets/img/eduroam/configure-networkmanager.png)
 
-Once these instructions are complete, you will be able to connect to the
-new eduroam setup without needing to run any scripts!
+Your computer should now be connected to the `eduroam` network.
+
+## CLI Setup
+
+Note: Both the `iwd` and `wpa_supplicant` configurations were tested on an otherwise-blank Arch install ISO. `nmcli` configuration was tested on Tails. File location may differ across distributions. If directories don't exist, check your distribution's documentation for the correct location of the files.
+
+### nmcli Configuration
+
+`nmcli` is the terminal interface for NetworkManager. This means that it can either use the `p12` or `pem` variant of your certificate and key. The instructions below are using the `p12` variant. If you wish to use the `pem` variant, replace all instances of the `p12` file with your `pem` file. This information is also useful for troubleshooting the GUI setup, should it work incorrectly.
+
+1. To find out what the name of your network interface is, run `nmcli`. This will return several results. For the examples that follow, this tutorial assumes that `wlan0` is the correct network interface.
+
+2. Open a terminal, and run the following command: 
+```
+nmcli connection add type wifi ifname wlan0 con-name eduroam ssid eduroam
+cd /etc/NetworkManager/system-connections/
+```
+This will add a new connection into NetworkManager's location on the drive, then move you into the directory where it was added. 
+
+3. Open the `eduroam.nmconnection` file in the editor of your choice *as root*. If you do not open it as root, the changes will not be saved.
 
-### wpa_supplicant (CLI) Setup
+4. Edit the file so that it contains the following lines
+```
+[connection]
+id=eduroam
+uuid=
+type=wifi
+interface-name=wlan0
+permissions=
+
```

Comment:
whatever is doing the formatting really hates config blocks with empty lines in it - which is why it got messed up the first time (maybe this is an option somewhere to be more true to github flavored markdown)

Reply:
Noted. Will go through and edit accordingly.